conpot/protocols/s7comm/s7_server.py

```diff
@@ -113,12 +112,13 @@ def handle(self, sock, address):
 
                         # will throw exception if the packet does not contain the S7 magic number (0x32)
                         S7_packet = S7().parse(cotp_base_packet.trailer)
-                        logger.debug('Received S7 packet: magic:{0} pdu_type:{1} reserved:{2} req_id:{3} param_len:{4} '
-                                     'data_len:{5} result_inf:{6}'.format(
+                        logger.debug(
+                            'Received S7 packet: magic:%s pdu_type:%s reserved:%s req_id:%s param_len:%s '
+                            'data_len:%s result_inf:%s session_id:%s',
```

Comment:
There is no need for that right ? Interpolation.  logging.debug(msg[, *args[, **kwargs]])